src/azure-cli/azure/cli/command_modules/monitor/operations/metric_alert.py

```diff
@@ -13,18 +13,30 @@
 def create_metric_alert(client, resource_group_name, rule_name, scopes, description, condition, disabled=False,
                         tags=None, actions=None, severity=2, window_size='5m', evaluation_frequency='1m',
                         auto_mitigate=None):
-    from azure.mgmt.monitor.models import MetricAlertResource, MetricAlertSingleResourceMultipleMetricCriteria
+    from azure.mgmt.monitor.models import MetricAlertResource, MetricAlertSingleResourceMultipleMetricCriteria, \
+        MetricAlertMultipleResourceMultipleMetricCriteria
```

Comment:
you can use from .. import (class1, class2, ...), so that "\" is no longer needed

Reply:
fixed